updates/37/updates.sql

```diff
@@ -1,5 +1,53 @@
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x8C130499D33097D4D000D3332E1672f75b431543', 'Hoppy', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x8C130499D33097D4D000D3332E1672f75b431543', 'A', 1, '0x8C130499D33097D4D000D3332E1672f75b431543', 8, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x8C130499D33097D4D000D3332E1672f75b431543', 'HOPPY', "hoppy-token", "HOPPY", NULL, 1689380339, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:56/erc20:0x90869b3a42E399951bd5F5fF278B8CC5ee1Dc0fE', 'REX', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:56/erc20:0x90869b3a42E399951bd5F5fF278B8CC5ee1Dc0fE', 'A', 56, '0x90869b3a42E399951bd5F5fF278B8CC5ee1Dc0fE', 18, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:56/erc20:0x90869b3a42E399951bd5F5fF278B8CC5ee1Dc0fE', 'REX', "revox", "REX", NULL, 1733297928, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('solana/token:4eDf52YYzL6i6gbZ6FXqrLUPXbtP61f1gPSFM66M4XHe', 'SOON', 'Y'); INSERT INTO solana_tokens(identifier, token_kind, address, decimals, protocol) VALUES('solana/token:4eDf52YYzL6i6gbZ6FXqrLUPXbtP61f1gPSFM66M4XHe', 'D', '4eDf52YYzL6i6gbZ6FXqrLUPXbtP61f1gPSFM66M4XHe', 9, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('solana/token:4eDf52YYzL6i6gbZ6FXqrLUPXbtP61f1gPSFM66M4XHe', 'SOONSOL', '', '', NULL, 1745739869, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('ZEUS', 'Zeus', 'P'); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('ZEUS', 'ZEUS', '', '', NULL, NULL, NULL);
+*
+INSERT INTO assets(identifier, name, type) VALUES('eip155:1/erc20:0x0f7dC5D02CC1E1f5Ee47854d534D332A1081cCC8', 'Pepes Dog', 'C'); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES('eip155:1/erc20:0x0f7dC5D02CC1E1f5Ee47854d534D332A1081cCC8', 'A', 1, '0x0f7dC5D02CC1E1f5Ee47854d534D332A1081cCC8', 9, NULL); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES('eip155:1/erc20:0x0f7dC5D02CC1E1f5Ee47854d534D332A1081cCC8', 'ZEUS', "pepes-dog", "", NULL, 1747863635, NULL);
```

Comment:
is this token different from the one above?

Reply:
no, different tokens